src/scripts/main.js

```diff
@@ -1,3 +1,226 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+
+let headers, tbody, rows;
```

Comment:
It is not a good practice to declare variables without assigning value to them, especially when they are global. It could lead to unexpected behavior. It's better to declare them in the scope where they are used.